### PR TITLE
Enhance the IdentitySearchList view

### PR DESCRIPTION
**Pull request by gsvr**
- [x] Filter against multiple parameters
- [x] Exclude inactive addresses when additional param is provided

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: identities/views.py [hunk 1]
```diff
@@ -101,14 +101,54 @@ def get_queryset(self):
...
+        if include_inactive is True:
+            return identities
+
+        else:
+            inactive_excluded = identities
+            for identity in identities:
+                for param in exclude_inactive:
+                    q_key = identity.details["addresses"][param[0]][param[1]]
+                    if ('inactive' in q_key and
+                       q_key['inactive'] in [True, 'True']):
```
Comment (imsickofmaps) [inline identities/views.py]: the indentation here looks screwy

File: identities/tests.py [hunk 2]
```diff
@@ -278,6 +278,59 @@ def test_read_identity_search_multiple(self):
...
+                "name": "Test Name 4",
+                "addresses": {
+                    "msisdn": {
+                        "+27123": {}
+                    }
+                }
+            }
+        })
+        # Execute
+        response_include_inactive = self.client.get(
```
Comment (imsickofmaps) [inline identities/tests.py:313]: Please add a query without the `include_inactive` param to validate the default setting also works

Comment (gsvr) [reply identities/tests.py:313]: Done

**Comment by imsickofmaps** [comment]
👍

**Pull request merged**